src/components/pageControl/index.js

```diff
@@ -206,8 +206,9 @@ class PageControl extends PureComponent {
     return (
       <View
         style={[styles.container, containerStyle]}
-        accessible
-        accessibilityLabel={`page control page ${this.props.currentPage + 1}`}
+        accessible={false}
+        accessibilityElementsHidden
```

Comment:
why aren't you using our `inaccessible` prop we added for View

Reply:
Fixed